server/src/test/java/io/deephaven/server/runner/DeephavenApiServerTestBase.java

```diff
@@ -40,8 +82,8 @@ public void setUp() throws Exception {
         logBuffer = new LogBuffer(128);
         LogBufferGlobal.setInstance(logBuffer);
 
-        serverComponent = DaggerDeephavenApiServerInProcessGroovyComponent
-                .builder()
+        serverComponent = DaggerDeephavenApiServerTestBase_TestComponent.builder()
+                .withExecutionContext(ExecutionContext.createForUnitTests())
```

Comment:
I see that `FlightTestModule` has a provider for this - should we have a `ExecutionContextUnitTestModule` that provides this instead?